gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -4,37 +4,52 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
+import React, { useState } from 'react';
+
+import { NxButton } from '@sonatype/react-shared-components';
 
 import HostedVersionsSelect from './HostedVersionsSelect';
+import DarkModeModal from '../DarkModeModal/DarkModeModal';
 
 const plaidVillain = require('./plaid-villain.svg');
 const tartanLogotype = require('./tartan-logotype.svg');
 
 import './PageHeader.scss';
 
-const PageHeader = () => (
-  <header className="nx-page-header gallery-page-header">
-    <div className="nx-page-header__inner">
-      <a className="gallery-page-header__home-link"
-         href="#/">
-        <img className="gallery-page-header__logo-image"
-             src={plaidVillain}
-             alt="" />
-        <img className="gallery-page-header__logotype-image"
-             src={tartanLogotype}
-             alt="Sonatype Tartan" />
-      </a>
-
-      <span className="gallery-page-header__name">
-        React Shared Component Library
-      </span>
-
-      <div className="gallery-page-header__version">
-        <HostedVersionsSelect />
+const PageHeader = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  return (
+    <header className="nx-page-header gallery-page-header">
+      <div className="nx-page-header__inner">
+        <a className="gallery-page-header__home-link"
+           href="#/">
+          <img className="gallery-page-header__logo-image"
+               src={plaidVillain}
+               alt="" />
+          <img className="gallery-page-header__logotype-image"
+               src={tartanLogotype}
+               alt="Sonatype Tartan" />
+        </a>
+
+        <span className="gallery-page-header__name">
+          React Shared Component Library
+        </span>
+        <NxButton className="nx-btn--open-theme-settings"
```

Comment:
Instead of a modifier `nx-btn--open-theme-settings` which implies changes in behaviour or state or is a variant of, and since this style is being used to adjust spacing within the page header, I wonder if we should change the class name to make it part of the page header:
Maybe something like `gallery-page-header__theme-settings-button` 🤔

Reply:
> Instead of a modifier `nx-btn--open-theme-settings` which implies changes in behaviour or state or is a variant of, and since this style is being used to adjust spacing within the page header, I wonder if we should change the class name to make it part of the page header: Maybe something like `gallery-page-header__theme-settings-button` 🤔

That's a great point and something I hadn't thought of. Renamed here: [6118e55](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/6118e559989abac6643ed4a9d36d2e06069061bb)